### DISCUSSION TITLE
I got an error and I don't know how to solve it

### DISCUSSION THREAD

**Discussion by RAFAKITL** [rupeshs/fastsdcpu#103] [Q&A]
![image](https://github.com/rupeshs/fastsdcpu/assets/118015647/f188b145-ce97-4ab8-8e93-fb25bdf8d9cc)
![image](https://github.com/rupeshs/fastsdcpu/assets/118015647/16f7f19e-657a-4d5f-9a5a-70ffcf1d4ada)

They happen when I run install.bat, then at the final it says that is install, but trying to open with start.bat, it just shut the window

**Comment by rupeshs** [comment]
@RAFAKITL  Can you share following details?
Operating system :
Python version :

**Comment by RAFAKITL** [comment]
Thank you so much, after checking the version of Python I found out that the libraries doesn't work with my version so I just install the version and it work again. Thank you
